src/Components/Activities/ActivitiesList.js

```diff
@@ -5,22 +5,32 @@ import {ACTIVITIES} from '../common/text/textActivity';
 import {getAllActivities} from '../../Services/activityService';
 import ActivitiesCards from './ActivitiesCards/ActivitiesCards';
 import ListPagination from '../common/ListPagination/ListPagination';
+import LoadingSpinner from "../../Utils/loadingSpinner";
 const ActivitiesList = () => {
 
     const [items, setItems] = useState([]);
+    const [loading, setIsLoading] = useState(true);
 
     useEffect(async() => {
-        getAllActivities()
-            .then(allActivities => setItems(allActivities));
+         const allActivities = await getAllActivities()
+            setItems(allActivities);
```

Comment:
delete extra spaces at the beginning of the function

Reply:
also do not delete the .then functionallity, please improve this code with something like:
```
useEffect(() => {
    getAllActivities()
        .then(allActivities => {
            setItems(allActivities);
            setIsLoading(false);
        })
},[])
```

Reply:
ready